### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by chrisbeckett8** [mlampros/textTinyR#11]
Hey, I tried to install your package, the vignette looks awesome by the way, but this is what it is giving me? Any help?

Thanks,
Chris

Using Cran yields: 

install.packages("tinyTextR")
Warning in install.packages :
  package ‘tinyTextR’ is not available (for R version 3.6.1)

Using github yields:

devtools::install_github('mlampros/textTinyR')
Downloading GitHub repo mlampros/textTinyR@master

Error: Failed to install 'textTinyR' from GitHub:
  (converted from warning) error 1 in extracting from zip file

**Comment by mlampros** [comment]
hi @chrisbeckett8,

there is a mistake in your command when you attempt to install from CRAN, the actual command should be

```R
install.packages('textTinyR')
```
About your 
```R

devtools::install_github('mlampros/textTinyR')
```
the error is not very informative but if you are on a Windows operating system you need the [Rtools](https://cran.r-project.org/bin/windows/Rtools/) installed. 
Please let me know if it worked.

**Comment by chrisbeckett8** [comment]
Ah I see my mistake. I downloaded the one from CRAN. Is there any advantage to downloading the github version?

Thanks,
Chris

**Comment by mlampros** [comment]
No, currently not because there are no differences between the github version and the uploaded package on CRAN. Actually, the download of the github version makes sense if there are updates that are not pushed to CRAN.

**Comment by chrisbeckett8** [comment]
Thanks. Hey could you update this blog post ?
http://mlampros.github.io/2018/04/04/extending_textTinyR_package/

Specifically, this function skipgram_cbow and all the control parameter are deprecated and have new names according to the latest install of fastTextR and Blog Post 1. 

vecs = fastTextR::skipgram_cbow(input_path = PATH_INPUT, output_path = PATH_OUT, 
                                method = "skipgram", lr = 0.075, lrUpdateRate = 100, 
                                dim = 300, ws = 5, epoch = 5, minCount = 1, neg = 5, 
                                wordNgrams = 2, loss = "ns", bucket = 2e+06,
                                minn = 0, maxn = 0, thread = 6, t = 1e-04, verbose = 2)

**Comment by mlampros** [comment]
hi @chrisbeckett8,

in this blog post I've included the following lines to inform users that the code was updated,

```
UPDATE 11-04-2019: There is an updated version of the fastText R package which includes 
all the features of the ported fasttext library. Therefore the old fastTextR repository is archived.
See also the corresponding blog-post.
```
web-link to the updated version of 'fastText': https://github.com/mlampros/fastText
web-link to the blog post about the updated version: http://mlampros.github.io/2019/04/11/fastText_updated_version/

**Comment by chrisbeckett8** [comment]
I did see that. Thank you. I think a lot of people are interested in clustering similar words. So I think if you had the actual code updated in the example on the blog post,, you would get more exposure for your package. But thank you for your help and nice job on it.

Thanks,
Chris
